### ISSUE TITLE
Bug 8781406: A11y_OneNoe Clipper_Verify accessibility with Change Clipper Mode_Keyboard: The keyboard focus moving to the close button from the 'Copied from link' instead of the remaining article section.

### ISSUE THREAD

**Issue by aanchalbhansali** [OneNoteDev/WebClipper#612]
Environment Details: 
Application Name: OneNote Clipper-Web
OS: Windows 11 OS build 26058.1000
Edge Dev Version: Version 123.0.2400.1 (Official build) dev (64-bit)
AT's: Keyboard
Pre-Requisites:
Open the URL: [OneNote Web Clipper Installation](https://www.onenote.com/clipper)
Invoke the 'Get OneNote web Clipper for Microsoft Edge' button.
Edge Add-ons page will appear. Invoke the Get button.
OneNote Clipper will be added in edge extension section. 
Expand the Extension button and turn on the OneNote Clipper Show in Toolbar button.
OneNote Clipper will appear beside of the search field.
Now invoke the OneNote clipper button.
Sign in with V-id.
Repro Steps:
Open the [Ankylosaurus - Wikipedia](https://en.wikipedia.org/wiki/Ankylosaurus)
Now invoke the OneNote clipper button.
OneNote Clipper dialog box will appear.
Tab to navigate to the 'Copied from link'.
Press tab key and observe the issue.
Actual Result:
The keyboard focus moving to the close button from the 'Copied from link' instead of the remaining article section.
Expected Result:
The keyboard focus should be move to the remaining article section. Should not move to the closed button.
User Impact:
Keyboard users will confuse, if the keyboard focus is moving to the close button from the 'Copied from link' instead of the remaining article section.
MAS Reference:
[MAS 2.4.3 – Focus Order](https://aka.ms/MAS2.4.3)
